### PR TITLE
Add easy functions

### PR DESCRIPTION
**Pull request by yvesfracari**
This PR:

Closes BLEU-116, BLEU-117, BLEU-118, BLEU-119, BLEU-120, BLEU-121, BLEU-122, BLEU-123 and BLEU-127.

### CODE REVIEW

File: balpy/balpy.py [hunk 1]
```diff
@@ -1543,6 +1548,50 @@ def balVaultGetInternalBalance(self, tokens, address=None):
...
+		address, specialization = vault.functions.getPool(poolId).call()
+		return address, specialization
+
+	def balVaultGetPoolTokenInfo(self, poolId, tokenAddress):
+		vault = self.balLoadContract("Vault")
+		tokenInfo = vault.functions.getPoolTokenInfo(poolId, tokenAddress).call()
+		cash = tokenInfo[0]
+		managed = tokenInfo[1]
+		lastChangeBlock = tokenInfo[2]
+		assetManager = tokenInfo[3]
```
Comment (ribeirojose) [inline balpy/balpy.py:1562]: could also be 
```suggestion
		cash, managed, lastChangeBlock, assetManager, *_ = tokenInfo
```

File: balpy/balpy.py [hunk 2]
```diff
@@ -1515,6 +1515,11 @@ def balVaultWeth(self):
 		wethAddress = vault.functions.WETH().call();
 		return(wethAddress);
 
+	def balVaultGetAuthorizer(self):
+		vault = self.balLoadContract("Vault")
```
Comment (ribeirojose) [inline balpy/balpy.py:1519]: I guess we're not talking about improvements here, but do we need to load the vault contract every time? Wouldn't it make sense to preload it to a `self.vault`?

**Review by ribeirojose — APPROVED** [review]

File: balpy/balpy.py (hunk 1 above)
Comment (yvesfracari) [reply balpy/balpy.py:1562]: Yes! My idea was to make clear to the user how to parse the output. Also, is the way that is done in the other methods. I wanted to keep this structure for now

File: balpy/balpy.py (hunk 2 above)
Comment (yvesfracari) [reply balpy/balpy.py:1519]: Agreed! We should keep it in mind when we refactor the lib

**Pull request merged**